src/components/molecules/subheaderLinks/SubheaderLinks.tsx

```diff
@@ -0,0 +1,30 @@
+import { ListSubheader, ListItemButton, ListItemText } from '@mui/material';
+import { NavStyle } from '../../../common/utils/newStyle';
+
+interface ListButtonProps {
+  label: string;
+  filter: string;
+}
+
+interface SubheaderLinksProps {
+  title: string;
+  links: ListButtonProps[];
+  navigate: (route: string) => void;
+}
+
+export function SubheaderLinks({ title, links, navigate }: SubheaderLinksProps) {
+  return (
+    <ListSubheader component="div" sx={{ ...NavStyle }}>
+      {title}
```

Comment:
🔧 I think we got rid of `title` in the latest mocks.

Reply:
Yeah I just overlooked this again.